CMakeLists.txt

```diff
@@ -13,12 +13,18 @@ if(POLICY CMP0015)
 	cmake_policy(SET CMP0015 OLD)
 endif()
 
+# allow relative paths in LINK_DIRECTORIES
+if(POLICY CMP0081)
+	cmake_policy(SET CMP0081 OLD)
+endif()
```

Comment:
This addition made it possible to specify relative `EXTRA_LINK` directories in addition to `EXTRA_INC`.